headerItems.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request');
```

Comment:
When the file is linted there will be a bunch of suggestions to update the syntax such as using `const` in place of `var`, using string interpolation rather than concatenation, arrow functions rather than function expressions, tweaks to the spacing, etc.

Reply:
Lint comments have been actioned.